.github/workflows/ci.yml

```diff
@@ -8,16 +8,20 @@ jobs:
       matrix:
         node-version: [8.x, 10.x, 12.x]
     steps:
-    - uses: actions/checkout@v1
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: ${{ matrix.node-version }}
-    - name: Install dependencies
-      run: yarn install --frozen-lockfile
-    - name: Run linter
-      run: yarn lint
-    - name: Run tests
-      run: yarn test
-    env:
-      CI: true
+      - uses: actions/checkout@v1
```

Comment:
Could we leave the indentation on the same level? There's no difference in yaml (the `-` is already treated like an indentation), and better to leave our action yaml looking like what's in the Github documentation :).

Reply:
My .yaml linter recommended using this indentation. If you look closely the Github Actions documentation is very inconsistent regarding array indentation, take a look at different sections in this page https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions

Looking at various sources seems that this indentation is more popular https://yaml.org/start.html https://docs.docker.com/compose/compose-file/

Reply:
Hmm good point. I spent a bit of time aligning them all to be on the same level today... but it looks like I should revert and make it consistent against the indented version.